hazelcast/src/test/java/com/hazelcast/config/replacer/AbstractPbeReplacerTest.java

```diff
@@ -40,7 +39,7 @@
  * Unit tests for {@link AbstractPbeReplacer}.
  */
 @RunWith(HazelcastParallelClassRunner.class)
-@Category({ QuickTest.class, ParallelJVMTest.class })
+@Category(QuickTest.class)
```

Comment:
I think you changed this because parallel runner and parallel jvm tests should not be set at the same time. Remember it from a slack discussion